ARCharts/ARBarChart+Bar.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ARBarChartBar.swift
+//  ARCharts
+//
+//  Created by Christopher Chute on 7/25/17.
+//  Copyright © 2017 Boris Emorine. All rights reserved.
+//
+
+import SceneKit
+import UIKit
+
+extension ARBarChart {
+    
+    public class Bar: SCNNode {
+        
+        public var series: Int = 0
```

Comment:
Agreed. I defaulted them as a hack to get around [this error](https://stackoverflow.com/questions/24021093/error-in-swift-class-property-not-initialized-at-super-init-call). Fixing it before merging.